### DISCUSSION TITLE
Domotizzare Citofono

### DISCUSSION THREAD

**Discussion by nilloi** [indomus/forum#2085] [GENERALE - Discussioni generiche]
Buongiorno a tutti , abito in un condominio e ho un citofono Urmet 1131 del quale allego foto dell'interno. Vorrei sapere come posso collegarlo a uno shelly in modo da ricevere una notifica quando qualcuno suona e poter anche aprire il portone. Inoltre esiste un metodo, con anche o senza shelly, per poter rispondere col telefono tramite una chiamata via whatsapp o telegram ovviamente appoggiandomi al wifi di casa e senza fare modifiche che alterino il suo uso "analogico" con la cornetta? Grazie 
![IMG_20221116_172002](https://user-images.githubusercontent.com/118290154/202243223-8022ed5d-32b2-4652-944b-ab99894b50ab.jpg)

**Comment by NdR91** [comment]
Da una BREVISSIMA ricerca su Internet, quindi per ora prendilo con le pinze, le uscite 10 e 9 dovrebbero essere quelle per aprire la porta.

Uno Shelly qui andrebbe già bene, ma se vuoi anche la notifica del campanello ti consiglio un D1 Mini + Relè per l'apertura, in modo da sfruttare i pin rimanenti del D1 per ricevere l'input del campanello. Ora non riesco a vedere da dove potresti prendere il campanello.

C'è da aggiungere anche un'altra cosa. Per Campanello + Conversazione, potresti guardare questo: [Ring Intercom](https://indomus.it/news/ring-chi-ce-al-citofono-amazon-presenta-intercom-la-soluzione-per-renderlo-smart/), quindi poi utilizzando lo Shelly per l'apertura sui Pin 10 e 9.

Escluderei a priori una modifica DYI per la parte "conversazione", tantomeno tramite chiamata Whatsapp o Telegram.
